app/lib/actions/katello/pulp_selector.rb

```diff
@@ -2,11 +2,29 @@ module Actions
   module Katello
     class PulpSelector < Actions::Base
       def plan(backend_actions, repository, smart_proxy, *args)
+        return_action_output = args.dig(0, :return_output) || false
+        found_action = PulpSelector.select(backend_actions, repository, smart_proxy)
+        fail "Could not locate an action for type #{backend_type}" unless found_action
+        if return_action_output
+          sequence do
+            action_output = plan_action(found_action, repository, smart_proxy, *args).output
+            plan_self(:output => action_output)
```

Comment:
can you rename this 'output' key to something more clear like, 'subaction_output' or something like that

Reply:
since actions already have an 'output', i'd like to keep that more clear.